dockerize.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+IMAGE_NAME="dashboard"
+DOCKER_HOME="/dashboard/"
+
+if [ "$1" == "init" ]; then
+    docker build -t $IMAGE_NAME .
+elif [ "$1" == "build" ]; then
+    docker run --rm --volume "$(pwd):$DOCKER_HOME" -ti $IMAGE_NAME yarn install --frozen-lockfile --no-bin-links

```

Comment:
`$(pwd)` doesn't work if you run the script from a different directory, that's pretty important for CI. 

`DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"`  sets `DIR` to the dir of the bash script regardless of where it's running from. 

More info https://stackoverflow.com/questions/59895/getting-the-source-directory-of-a-bash-script-from-within

Reply:
Not sure why it wasn't working on Windows before but it is working now so I am using this now!